js/main.js

```diff
@@ -0,0 +1,118 @@
+const pictureCount = 25;
+const avatarMinimalCount = 1;
+const avatarMaximalCount = 6;
+const likeMinimalCount = 15;
+const likeMaximalCount = 200;
+const commentMinimalCount = 1;
+const commentMaximalCount = 10;
+
+const names = [
+  'Лиза',
+  'Ангелина',
+  'Кристина',
+  'Саша',
+  'Женя',
+  'Олег',
+  'Лола',
+  'Даша',
+  'Петя',
+  'Даня',
+  'Ника',
+  'Эвелина',
+  'Лев',
+  'Миша',
+  'Паша',
+  'Сережа',
+  'Коля',
+  'Настя',
+  'Аня',
+  'Надя',
+  'Карина',
+  'Зоя',
+  'Эмили',
+  'Николь',
+  'Милана'
+];
+
+const descriptions = [
+  'Тост с беконом',
+  'Моя собака',
+  'Вид из окна',
+  'Мои подружки',
+  'Кукурузный суп-пюре',
+  'Путешествия 2020',
+  'Токио и Сеул',
+  'Цели 2023',
+  'Маникюр',
+  'Розовый - хит сезона',
+  'Жизнь без глютена',
+  'Жизнь без глюкозы',
+  'Жизнь без удовольствий',
+];
+
+const messages = ['Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'];
+
+
+const getRandomPositiveInteger = function (a, b) {
```

Comment:
```suggestion
const getRandomPositiveInteger = (a, b) => {
```

Кажется здесь будет достаточно стрелочной функции - поправь, пожалуйста, везде))
https://up.htmlacademy.ru/javascript/28/criteries#d5